content/doc/developer/publishing/wiki-page.adoc

```diff
@@ -36,6 +36,11 @@ There are the following migration steps:
 . Modify the URL documentation page reference in the project file so that it points to GitHub (link:/doc/developer/publishing/documentation/#referencing-the-documentation-page-from-the-project-file[documentation]).
 . Commit changes, push them to your fork and create a pull request against the repository.
 
+NOTE: After migration, the continued existence of out-of-date wiki pages causes a number of problems, such as duplication, confusion, and unclear migration progress. 
+This persistence not only makes the process more difficult overall, but it also makes it harder to identify which pages have already been migrated and which are still waiting. 
+This is a major technical obstacle in the way of content migration from the wiki. 
+When migrating documentation, after the material has been properly moved, the out-of-date wiki page must be removed and a redirect must be configured to the migrated version. 
```

Comment:
```suggestion
NOTE: After migration, the continued existence of out-of-date wiki pages can cause several problems, such as duplication, confusion, and unclear migration progress. 
This persistence makes the process more difficult overall and makes it harder to identify which pages have already migrated and those still waiting for migration. 
This is a major technical obstacle for content migration from the wiki. 
When migrating documentation, after the material has been properly moved, you must remove the out-of-date wiki page and configure a redirect to the migrated version.  
```
I have provided an updated suggestion, as this is a bit better than what is currently written/what I had suggested previously.